resources/services/ssm/associations.go

```diff
@@ -0,0 +1,319 @@
+			},
+			{
+				Name:          "status_additional_info",
+				Description:   "A user-defined string.",
+				Type:          schema.TypeString,
+				Resolver:      schema.PathResolver("Status.AdditionalInfo"),
+				IgnoreInTests: true,
+			},
+			{
+				Name:          "sync_compliance",
+				Description:   "The mode for generating association compliance",
+				Type:          schema.TypeString,
+				IgnoreInTests: true,
+			},
+		},
+		Relations: []*schema.Table{
+			{
+				Name:          "aws_ssm_association_target_locations",
+				Description:   "The combination of Amazon Web Services Regions and Amazon Web Services accounts targeted by the current Automation execution.",
+				Resolver:      fetchSsmAssociationTargetLocations,
+				IgnoreInTests: true,
+				Columns: []schema.Column{
+					{
+						Name:        "association_cq_id",
+						Description: "Unique CloudQuery ID of aws_ssm_associations table (FK)",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.ParentIdResolver,
+					},
+					{
+						Name:        "accounts",
+						Description: "The Amazon Web Services accounts targeted by the current Automation execution.",
+						Type:        schema.TypeStringArray,
+					},
+					{
+						Name:        "execution_role_name",
+						Description: "The Automation execution role used by the currently running Automation",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "regions",
+						Description: "The Amazon Web Services Regions targeted by the current Automation execution.",
+						Type:        schema.TypeStringArray,
+					},
+					{
+						Name:        "target_location_max_concurrency",
+						Description: "The maximum number of Amazon Web Services Regions and Amazon Web Services accounts allowed to run the Automation concurrently.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "target_location_max_errors",
+						Description: "The maximum number of errors allowed before the system stops queueing additional Automation executions for the currently running Automation.",
+						Type:        schema.TypeString,
+					},
+				},
+			},
+			{
+				Name:        "aws_ssm_association_targets",
+				Description: "An array of search criteria that targets instances using a key-value pair that you specify",
+				Resolver:    fetchSsmAssociationTargets,
+				Columns: []schema.Column{
+					{
+						Name:        "association_cq_id",
+						Description: "Unique CloudQuery ID of aws_ssm_associations table (FK)",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.ParentIdResolver,
+					},
+					{
+						Name:        "key",
+						Description: "User-defined criteria for sending commands that target instances that meet the criteria.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "values",
+						Description: "User-defined criteria that maps to Key",
+						Type:        schema.TypeStringArray,
+					},
+				},
+			},
+		},
+	}
```

Comment:
not sure but maybe can we change into JSON field

Reply:
I agree. Collapsed into a JSON field.